### PR TITLE
feat: Delete rooms in the device connect fragment

### LINKED ISSUE #2287: Add feature to delete room
**Describe the feature you'd like**
Add the feature to delete room

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #2287

Changes: 
User can now delete the name of the rooms he added before

Screenshots for the change: 
ezgif com-video-to-gif

Apk: 
[app-fdroid-debug.apk.zip](https://github.com/fossasia/susi_android/files/3341963/app-fdroid-debug.apk.zip)

### CODE REVIEW

**Comment by atm1504** [comment]
@iamareebjamal please review it.

**Comment by iamareebjamal** [comment]
PRF

**Comment by atm1504** [comment]
@liveHarshit can you please review it. : )

**Comment by liveHarshit** [comment]
> @liveHarshit can you please review it. : )

IMO alert dialog should display.

**Comment by atm1504** [comment]
> > @liveHarshit can you please review it. : )
> 
> IMO alert dialog should display.

We are approaching in the way present in google home. There no alert box is shown. So we have not used it here.

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -137,4 +138,8 @@
     <dimen name="height_minimum">2dp</dimen>
     <dimen name="height_room_recycler">20dp</dimen>
     <dimen name="width_room_recycler">20dp</dimen>
+    <dimen name="room_image_weight">1</dimen>
+    <dimen name="room_name_weight">4</dimen>
+    <dimen name="room_delete_weight">1</dimen>
```
Comment (liveHarshit) [inline app/src/main/res/values/dimens.xml]: No need to declare weight here, these are not dimensions.

Comment (atm1504) [reply app/src/main/res/values/dimens.xml]: Updated. Please review now

**Comment by atm1504** [comment]
@iamareebjamal please review it now

**Pull request merged**